.github/workflows/ci.yml

```
@@ -23,10 +23,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
-        os: [ubuntu-latest]
-        python-version: ["3.11"]
-    timeout-minutes: 60
+        os: [ubuntu-latest, macos-15, windows-latest]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```
py3.8 is over its life-cycle